libs/shared/ui/src/lib/components/chart/chart-utils.ts

```diff
@@ -74,3 +74,29 @@ export function createXAxisConfig(
     interval: 'preserveStartEnd',
   }
 }
+
+// Helper function to format timestamp for display
+export function formatTimestampForDisplay(timestamp: number | string, useLocalTime: boolean): string {
```

Comment:
Could you use date an util from `util-dates.ts` to avoid having a custom function only for this element ?